src/Console/Command/StageCommand.php

```diff
@@ -3,21 +3,72 @@
 namespace PhpTuf\ComposerStager\Console\Command;
 
 use PhpTuf\ComposerStager\Console\Misc\ExitCode;
+use PhpTuf\ComposerStager\Domain\Stager;
 use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class StageCommand extends Command
 {
     protected static $defaultName = 'stage';
 
+    /**
+     * @var \PhpTuf\ComposerStager\Domain\Stager
+     */
+    private $stager;
+
+    public function __construct(Stager $stager)
+    {
+        parent::__construct(static::$defaultName);
+        $this->stager = $stager;
+    }
+
+    /**
+     * @throws \Symfony\Component\Console\Exception\InvalidArgumentException
+     */
     protected function configure(): void
     {
-        $this->setDescription('Stages a Composer command');
+        $this
+            ->setDescription('Stages a Composer command')
```

Comment:
Not a big deal at this stage of things, but this isn't very descriptive. Maybe "Stages a Composer command to run in another directory" would be clearer?

Reply:
What about "executes a Composer command in the staging directory"?